about.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About</title>
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+    integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+  <link rel="stylesheet" href="css/reset.css" media="screen">
+  <link rel="stylesheet" href="css/style.css" media="screen">
+  <link rel="stylesheet" media="screen and (max-width: 768px)" href="css/mobile_style.css">
+  <link rel="stylesheet" media="screen and (min-width: 769px)" href="css/desktop_style.css">
+</head>
+
+<body>
+  <header>
+
+    <nav>
+      <div class="mobileMenu">
+        <i class="fas fa-bars open"></i>
+      </div>
+
+      <div class="desktopMenu">
+        <ul class="social-links">
+          <li><i class="fab fa-facebook-f"></i></li>
+          <li><i class="fab fa-twitter"></i></li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Logout</li>
+        </ul>
+
+        <div class="navigation">
+          <p><i class="fas fa-times close"></i></p>         
+          <a href="index.html"><img src="assets/images/summit_logo.png" alt="Summit's logo" class="summitLogo"></a>
+          
+          
+          <ul class="menu">
+            <li class="menu-link"><a href="about.html">About</a></li>
```

Comment:
## [OPTIONAL]

- [ ] As we are on the about page already I think we don't really need to have an `About` ink in the navigation but we can need a link back to the home page. Kindly I would suggest you change this `About` link with a `Home` link

Reply:
Thank you @andersonlebon.  Suggested change implemented.